gitlab.go

```diff
@@ -437,31 +439,50 @@ type ListOptions struct {
 // NewClient returns a new GitLab API client. If a nil httpClient is
```

Comment:
There is no `httpClient` argument anymore.

Reply:
Updated the comment 👍